app/Modules/Group/Events/GroupEvent.php

```diff
@@ -66,7 +66,10 @@ public function getTopic(): string
      */
     public function shouldPublish(): bool
     {
-        return $this->group->isEp;
+
+       return $this->group->isEp || $this->group->isWg || $this->group->isCdwg;
```

Comment:
I think the reason this is not sufficient is elucidated by tracing the code path of what happens when a member is added to a group (regardless of whether EP, WG, CDWG):

A POST is made to `/api/groups/{group:uuid}/members`, which you can see in `app/Modules/Group/routes/api.php` (or perhaps more easily by looking at the output of `php artisan route:list`) is associated to `App\Modules\Group\Actions\MemberAdd`.

`handle` of that controller dispatches a `MemberAdded` event. This ultimately inherits from the `GroupEvent` you've modified here, but through `GroupMemberEvent` which overrides `shouldPublish` to include conjunction with `$this->group->expertPanel->definitionIsApproved;`

So even though you've modified `GroupEvent` to produce events for groups that are not EPs, but the subclass `GroupMemberEvents` that aren't from EPs cannot satisfy this conjunction, so do not end up being published to the data exchange.